homeassistant/components/matter/event.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             clusters.FixedLabel.Attributes.LabelList
         ):
             for label in labels:
-                if label.label == "Label":
+                if label.label.lower() == "label" or label.label.lower() == "button":
```

Comment:
I'm pretty sure that these are nullable values so this will crash if that happens

Reply:
I think you are right and, at the same time, also realized my solution does not really address the fundamental problem - Matter doesn't say what the label should be and trying to add more label variations could just goes on forever.  A simpler and better approach may be to keep the code with just the label.label == "Label", as it already is, but if there is no match for label.label == "Label", and labels is not empty, just select the first item in the "labels" list (its almost certain to be better than ignoring all the available label choices!).